docs/sources/companion-plugins/google-drive.mdx

```diff
@@ -94,6 +94,8 @@ Create a project for your app if you don’t have one yet.
 - On the project’s dashboard,
   [enable the Google Drive API](https://developers.google.com/drive/api/v3/enable-drive-api).
 - [Set up OAuth authorization](https://developers.google.com/drive/api/v3/about-auth).
+    - Under scopes, add the `https://www.googleapis.com/auth/drive.readonly` Drive API scope.
+    - Due to this being a sensitive scope Google will ask you to verify your app which can take up to 8 weeks.
```

Comment:
Not sure if this needs to explicitly mention this is solely for self-hosted Companion?